include/io.h

```diff
@@ -51,6 +51,12 @@ class IO {
   //! Write initial element volumes
   void write_volumes(const std::map<unsigned, double>& volumes);
 
+  //! Return .vtk file for stress
```

Comment:
Can we have one function that takes an argument be in `volume`, `stresses`, or `mesh` and return the path instead of three different functions?